### ISSUE TITLE
What format is the test set placed in tests/test-data?

### ISSUE THREAD

**Issue by McGaosai** [fizyr/keras-retinanet#961]
Please make sure that you follow the steps below when creating an issue.
Only use GitHub issues for issues with the implementation, not for issues with specific datasets or general questions about functionality.
If your issue is an implementation question, please ask your question on the #keras-retinanet Slack channel instead of filing a GitHub issue.
You can find directions for the Slack channel here: https://github.com/fizyr/keras-retinanet#discussions

Thank you!

1. Check that you are up-to-date with the master branch of keras-retinanet.
2. Check that you are up-to-date with the latest version of Keras: https://github.com/keras-team/keras.
3. Check that you are up-to-date with the latest version of TensorFlow.
   The installation instructions can be found here: https://www.tensorflow.org/get_started/os_setup.
4. Check that you have read the entire README.md: https://github.com/fizyr/keras-retinanet/README.md.
   Most noticably the FAQ section shows common issues: https://github.com/fizyr/keras-retinanet#faq.
5. Clearly describe the issues you're having including the expected behaviour, the actual behaviour
   and the steps required to trigger the issue.
6. Include relevant output from the commands you're executing, including full stack traces where relevant.
7. Remove this entire message and replace it with your issue.

**Comment by martinzlocha** [comment]
When creating issues please follow the steps above please. Mainly step 7!

To address the question in the title. There are several different formats, there is some data which follows the csv format and then there is a subset of the coco dataset etc. If you want to create your own dataset I would recommend you use the csv format. You can learn more about that in the readme.

**Comment by hgaiser** [comment]
@martinzlocha has answerred the question, closing this issue.
